arelle/CntlrCmdLine.py

```diff
@@ -302,6 +302,8 @@ def parseAndRun(args):
     parser.add_option("--formularunids", action="store", dest="formulaRunIDs", help=SUPPRESS_HELP)
     parser.add_option("--formulaCompileOnly", action="store_true", dest="formulaCompileOnly", help=_("Specify formula are to be compiled but not executed."))
     parser.add_option("--formulacompileonly", action="store_true", dest="formulaCompileOnly", help=SUPPRESS_HELP)
+    parser.add_option("--formulaCacheSize", action="store", dest="formulaCacheSize", help=_("Specify the number of fact aspect combinations to cache during formula evaluations. Negative numbers have no limit. (10_000_000 is default)"))
+    parser.add_option("--formulacachesize", action="store", dest="formulaCacheSize", help=_("Specify the number of fact aspect combinations to cache during formula evaluations. Negative numbers have no limit. (10_000_000 is default)"))
```

Comment:
Alternatively because this seems confusing, is there any reason to not just pass multiple flags and call `add_option` once instead?

```bash
>>> import optparse
>>> parser = optparse.OptionParser()
>>> parser.add_option('--integers', '--Integers', type=int)
<Option at 0x10577f190: --integers/--Integers>
>>> parser.parse_args(['--integers', "1"])
(<Values at 0x1057abd10: {'integers': 1}>, [])
>>> parser.parse_args(['--Integers', "1"])
(<Values at 0x1057aba90: {'integers': 1}>, [])
```

Also seems confusing to not use the library to handle the default:

```python
>>> parser.add_option('--integers', '--Integers', type=int, default=100000)
<Option at 0x101802210: --integers/--Integers>
>>> parser.parse_args([])
(<Values at 0x10159b710: {'integers': 100000}>, [])
```

Reply:
We also have to provide a default for the GUI and Python library paths